app/ingest_raw/README.md

```diff
@@ -0,0 +1,6 @@
+
+# ingest_raw
+
+This service is responsible for taking raw job description data and storing it in an object store. For now, this is a public S3 bucket. This service is responsible for knowing the details of where we store data in its rawest form.
```

Comment:
nit - this is a private S3 bucket already, no? You only enabled access to sam and me..

Reply:
oh true, you are correct